### PR TITLE
meta: dynamic versioning

### PR DESCRIPTION
**Pull request by lengau**
- [x] Have you followed the guidelines for contributing?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] Have you successfully run `tox`?

-----

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,3 +136,6 @@ dmypy.json
 
 # direnv
 .envrc
+
+# Ignore version module generated by setuptools_scm
```
Comment (tigarmo) [inline .gitignore:140]: this one is to satisfy black & ruff

File: craft_archives/__init__.py [hunk 2]
```diff
@@ -14,3 +14,17 @@
 # You should have received a copy of the GNU General Public License along
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 """craft-archives package demo."""
+
```
Comment (tigarmo) [inline craft_archives/__init__.py:17]: black-formatted this, and added the no-cover

**Review by tigarmo — APPROVED** [review]
Thanks for catching this! I had to make some changes, please take a look

**Comment by lengau** [comment]
Thank you @tigarmo ! Looks great. I'll merge this now and then whenever you're ready you can release 1.0.0

**Pull request merged**